infrastructure-playbooks/shrink-osd-ceph-disk.yml

```diff
@@ -171,6 +171,14 @@
         - "{{ osd_to_kill_disks.results }}"
         - "{{ osd_hosts }}"
 
+    - name: set_fact _osd_hosts
+      set_fact:
+        _osd_hosts: "{{ _osd_hosts | default([]) + [hostvars[item.0]['inventory_hostname']] }}"
+      with_together:
```

Comment:
I think we need to use the nested lookup instead of together otherwise we will have none value due to the length difference betweend groups.get(osd_group_name) and osd_hosts.